gpu/test_gpu.py

```diff
@@ -9,9 +9,10 @@
 class NvidiaGpuDriverTestCase(DataprocTestCase):
   COMPONENT = "gpu"
   INIT_ACTIONS = ["gpu/install_gpu_driver.sh"]
-  GPU_L4="type=nvidia-l4"
-  GPU_T4="type=nvidia-tesla-t4"
-  GPU_V100 = "type=nvidia-tesla-v100"
+  GPU_L4   = "type=nvidia-l4"
+  GPU_T4   = "type=nvidia-tesla-t4"
+  GPU_P100 = "type=nvidia-tesla-p100" # not available in us-central1-a
```

Comment:
we can remove P100 test cases since spark-rapids drop pascal since v2402, more details please find https://nvidia.github.io/spark-rapids/docs/archive.html#release-notes-3